gax-grpc/src/main/java/com/google/api/gax/grpc/GrpcDirectStreamController.java

```diff
@@ -99,6 +100,35 @@ void start(RequestT request) {
     }
   }
 
+  ApiStreamObserver<RequestT> startBidiStreaming() {
```

Comment:
Are we going to keep using `ApiStreamObserver` for the bidi case, even though we moved to `ResponseObserver` for the server streaming case?